rust/Makefile

```
@@ -2,7 +2,7 @@
 CRATES_TEST = trezor-thp trezor-tjpgdec
 CRATES = ${CRATES_TEST} trezor-client
 
-check: clippy test
+check: clippy test audit
```

checkmake (0.2.2)

[warning] Target "check" should be declared PHONY.

(phonydeclared)